examples/docking-protein-protein/docking-protein-protein-cltsel-test.cfg

```diff
@@ -31,25 +31,31 @@ hise_1 = 15
 [rigidbody]
 tolerance = 20
 ambig_fname = "data/e2a-hpr_air.tbl"
-sampling = 20
+sampling = 100
```

Comment:
This is a `test.cfg` made to execute fast. Isn't `sampling = 100` too much?

Reply:
Rigid body is fine - fast anyway - and for the cluster selection you need enough models